### PR TITLE
Class 4 api homework

### PR DESCRIPTION
**Pull request by tylerswartz**
submitting my api homework. One thing I wasn't able to figure out is how to validate the input from the user that it is an integer. I found a statement `input.is_a?Integer == true` that verifies it, but I don't understand how to put that in a loop that would keep asking the user for input until it is correct.

### CODE REVIEW

**Comment by tylerswartz** [comment]
I also added my secret number hw to the pull request.

File: tylerswartz/api01/api_output.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#required
+require 'json'
+require 'rest-client'
+
+#ask for zipcode
+puts 'This is outputed information from the Google Maps API'
+puts 'What US zipcode would you like to look up?'
+
+zipcode = gets.chomp
+
```
Comment (dladowitz) [inline tylerswartz/api01/api_output.rb]: To answer your question about validating zip codes you could do something like this.

gets.chomp returns a string, then I use `.to_i` to try and turn the string into an integer. 
Something like "5" will turn into the integer 5. A letter will just turn into 0. So I'm checking to make sure we don't have a 0 set in zipcode. 

```ruby
while zipcode.to_i == 0
  puts "Thats not a Zipcode. Please try again"
  zipcode = gets.chomp
end
```

File: tylerswartz/secret_number_v1.rb [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+			return 'low'
+		else
+			return 'high'
+		end
+	end
+end
+
+#Method to keep track of the number of guesses left and exit the game if they reach 3.
+def num_guess_check(y,secret_number)
+	if y == 3
```
Comment (dladowitz) [inline tylerswartz/secret_number_v1.rb]: I'd use a more descriptive name instead of `y`. 
Something like `attempts` or `guesses`

**Comment by dladowitz** [comment]
Great work creating reusable methods for the API homework

:+1:

**Pull request merged**